packages/@sanity/default-layout/src/navbar/loginStatus/LoginStatus.tsx

```diff
@@ -1,97 +1,35 @@
-
-import styles from './LoginStatus.css'
-
-interface MenuItem {
-  action: string
-  icon: React.ComponentType<Record<string, unknown>>
-  title: string
-}
+import {UserAvatar} from '@sanity/base/components'
+import {Button, MenuItem, Menu, MenuButton, Flex, Box, Text} from '@sanity/ui'
+import {ChevronDownIcon, LeaveIcon} from '@sanity/icons'
 
 interface LoginStatusProps {
   onLogout: () => void
 }
 
-interface LoginStatusState {
-  isOpen: boolean
-}
-
-export default class LoginStatus extends React.PureComponent<LoginStatusProps, LoginStatusState> {
-  state = {isOpen: false}
-
-  handleUserMenuItemClick = (item: MenuItem) => {
-    if (item.action === 'signOut') {
-      this.props.onLogout()
-    }
-
-    this.setState({isOpen: false})
-  }
-
-  handleButtonClick = () => {
-    this.setState((state) => ({isOpen: !state.isOpen}))
-  }
-
-  handleClose = () => {
-    this.setState({isOpen: false})
-  }
-
-  render() {
-    const menuItems: MenuItem[] = [
-      {
-        title: `Sign out`,
-        icon: IconSignOut,
-        action: 'signOut',
-      },
-    ]
-
-    const popoverContent = (
-      <div className={styles.menuWrapper}>
-        <Menu onAction={this.handleUserMenuItemClick} items={menuItems} />
-      </div>
-    )
-
-    return (
-      <ClickOutside onClickOutside={this.handleClose}>
-        {(ref) => (
-          <button
-            className={styles.root}
-            onClick={this.handleButtonClick}
-            ref={ref as React.Ref<HTMLButtonElement>}
-            title="Toggle user menu"
-            type="button"
-          >
-            <div className={styles.inner} tabIndex={-1}>
-              <LegacyLayerProvider zOffset="navbarPopover">
-                <Popover
-                  content={popoverContent as any}
-                  open={this.state.isOpen}
-                  placement="bottom-end"
-                >
-                  <div className={styles.avatarContainer}>
-                    <UserAvatar size="medium" tone="navbar" userId="me" />
-                  </div>
-                </Popover>
-              </LegacyLayerProvider>
-
-              <div className={styles.iconContainer}>
+export function LoginStatus({onLogout}: LoginStatusProps) {
+  return (
+    <MenuButton
+      button={
+        <Button mode="bleed" paddingX={1} paddingY={0} title="Toggle user menu">
+          <Flex align="center">
+            <UserAvatar size="medium" tone="navbar" userId="me" />
+            <Box marginLeft={1}>
+              <Text size={1}>
                 <ChevronDownIcon />
-              </div>
-            </div>
-
-            {this.state.isOpen && <Escapable onEscape={this.handleClose} />}
-          </button>
-        )}
-      </ClickOutside>
-    )
-  }
+              </Text>
+            </Box>
+          </Flex>
+        </Button>
+      }
+      id="login-status-menu"
+      menu={
+        <Menu>
+          <MenuItem text="Sign out" padding={4} icon={LeaveIcon} onClick={onLogout} />
```

Comment:
This should be `iconRight={LeaveIcon}`.

Reply:
Also, this menu item should have default padding (3), so `padding={4}` should be removed.